usgs-details.html

```diff
@@ -182,35 +180,55 @@
         }
 
         // Create header text for live and fallback values
-        function renderHeader(metric, isFallback) {
-            var firstWord = isFallback ? 'Typical ' : 'Current ';
-            return firstWord + metric.display + ' in the Schuylkill River:';
+        function renderHeader(metric, isFallback, isBackupSensor) {
+            if (isFallback) {
+                return 'Typical ' + metric.display + ' in the Schuylkill River:';
+            } else if (isBackupSensor) {
```

Comment:
Since the body of the `if` is a simple `return`, prefer an independent `if` to an `else if`.

```js
if (isFallback) {
  return ...;
}

if (isBackupSensor) {
  return ...;
}

return ...;
```